api/sync.js

```diff
@@ -0,0 +1,11 @@
+const settingsConfig = require('../config').settings;
```

Comment:
will set a new interval on every call to the API, so no need to expose this through an API, just start the interval when booting up the app for the first time.